StantUiIosLibrary/Classes/Tables/GroupSelectionTableViewWithCollapsedSearchBar/GroupSelectionTableViewWithCollapsedSearchBar.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  GroupSelectionTableViewWithCollapsedSearchBar.swift
+//  StantUiIosLibrary
+//
+//  Created by Mac Mini Novo on 29/07/19.
+//
+
+import UIKit
+
+public class GroupSelectionTableViewWithCollapsedSearchBar: UIView {
+    
+    internal var tableView: GroupSelectionTableView?
+    internal var searchView: DefaultSearchBar?
+    private var emptyMessageLabel: UILabel?
+    
+    var tableViewDelegate: GroupSelectionTableViewDidSelectDelegate?
```

Comment:
This delegate should have a weak reference

Reply:
Nice, man. I'm going to changed it on the other table view as well :)